episodes/07-infrastructure.md

```diff
@@ -48,7 +48,7 @@ this has been how The Carpentries presents its lesson websites to the world.
 
 ## Using The Carpentries Workbench
 
-Carpentries lesson websites are built with **The Carpentries Workbench**,
+Carpentries lesson websites are built with [**The Carpentries Workbench**](https://carpentries.github.io/sandpaper-docs/),
```

Comment:
```suggestion
Carpentries lesson websites are built with [**The Carpentries Workbench**](https://carpentries.github.io/workbench/),
```

I think this is the preferred target for Workbench links now.

Reply:
Great - I was not away and could not google it out for some reason.